plasmapy/physics/distribution.py

```diff
@@ -0,0 +1,85 @@
+"""Functions to deal with distribution : generate, fit, calculate"""
+
+
+from astropy import units
+
+from astropy.units import (UnitConversionError, UnitsError, quantity_input,
+                           Quantity)
+
+from ..constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi, e)
+from ..atomic import (ion_mass, charge_state)
+
+import numpy as np
+
+from ..utils import _check_quantity, check_relativistic, check_quantity
+
+
+def Maxwellian_1D(v,T,part ="e", V_drift = 0* units.m/units.s):
+    r"""Return the probability at the velocity `v` in m/s to find a particle `part` in a plasma of temperature `T` following the Maxwellian distribution function
+
+    Parameters
+    ----------
+    v: Quantity
+        The velocity in units convertible to m/s
+
+    T: Quantity
+        The temperature in Kelvin
+
+    part: string, optional
+        Representation of the ion species (e.g., 'p' for protons, 'D+'
+        for deuterium, or 'He-4 +1' for singly ionized helium-4),
+        which defaults to electrons.
+
```

Comment:
We could probably use a latex (mathrm) representation of the formula. I can get that done for you.

Come to think of it, AFAIK SymPy can autogenerate those representations... #136 ? :D

Reply:
I'm not so sure to understand this comment. I tried something tell me if this is what you think of.

Reply:
My apologies, I guess I wasn't clear at all here! I added a commit to show what I mean (the actual formula for the Maxwellian distribution) within the docstring.